app/services/invoice_builder.rb

```diff
@@ -1,32 +1,39 @@
 # frozen_string_literal: true
 
 class InvoiceBuilder
```

Comment:
this class is not yet used anywhere right? Could we add a button to be able to generate some invoices to test?

Reply:
@lukaseisenring wird die Rechnungen per Konsole generieren, dann braucht es den Knopf nicht (noch nicht)